client/components/DataManagement/DataManagementRow/index.jsx

```diff
@@ -425,12 +425,53 @@ const DataManagementRow = ({
             return otherVersionsInProgress.length;
         };
 
-        if (recommendedTestPlanVersions.length) {
-            const { earliestVersion, earliestVersionDate } = getVersionData(
-                recommendedTestPlanVersions,
+        /**
+         * Uses the truthy state of defined dates for a TestPlanVersion to
+         * determine if the version was ever in that phase at any given point.
+         * Useful if the phase of the TestPlanVersion is currently DEPRECATED.
+         * @param {"RECOMMENDED"|"CANDIDATE"|"DRAFT"} phase
+         * @returns {TestPlanVersion[]}
+         */
+        const deriveVersionsByDates = phase => {
+            switch (phase) {
+                case 'RECOMMENDED':
+                    return testPlanVersions.filter(
```

Comment:
I verified that this is working, and that's the important thing, but I was wondering, looking down at the query below, how does this work when the testPlanVersions variable apparently doesn't include deprecated versions?

Reply:
Thanks for clarifying offline.